tests/unit/test_endpoint.py

```diff
@@ -22,21 +22,21 @@ def test_filter(self):
             test = test_obj.filter(test="test")
             self.assertEqual(len(test), 2)
 
-    def test_filter_empty_kwargs(self):
```

Comment:
Neither this nor test_filter_reserved_kwargs makes sense with this PR, so I've lazily converted them to another type of test..